### ISSUE TITLE
No public API, available in swift pod

### ISSUE THREAD

**Issue by remirobert** [CocoaPods/CocoaPods#3837]
I created a swift pod, for iOS.
=> https://github.com/remirobert/TextDrawer

When i pod install this pod from a swift project with a PodFile and :
```
use_frameworks!
pod 'TextDrawer'
```

I can import the framework the framework, but all my classes are not available. I can't use my code.
I check the Pod repertory, and all the code is here.

How can i fix it.

This is my podscep : 

```
Pod::Spec.new do |s|
  s.name                  = "TextDrawer"
  s.version               = "1.0.1"
  s.summary               = "TextDrawer, is a UIView allows you to add text, with gesture, on UIView, or UIImage."
  s.requires_arc          = true
  s.homepage              = "https://github.com/remirobert/TextDrawer"
  s.ios.deployment_target = '8.0'
  # s.screenshots  = "www.example.com/screenshots_1.gif", "www.example.com/screenshots_2.gif"

  s.license               = "MIT"

  s.social_media_url      = 'https://twitter.com/remi936'
  s.author                = { "rémi " => "remirobert33530@gmail.com" }

  s.source                = { :git => "https://github.com/remirobert/TextDrawer.git", :tag => "1.0.1" }
  s.source_files = 'TextDrawer/*.swift'
  s.dependency 'Masonry'
end
```

Thanks :+1:

**Comment by orta** [comment]
https://github.com/remirobert/TextDrawer/tree/1.0.1/TextDrawer

Your 1.0.1 tag has no Swift files, only the master branch does, I'd recommend either changing the tag, or making a new version now with the new files.

**Comment by remirobert** [comment]
Thanks for your help. I fix it, with a new Tag called 1.0.2.
The files are there now : https://github.com/remirobert/TextDrawer/tree/1.0.2/TextDrawer
But i can't use my code in the example project.

<img width="602" alt="screen shot 2015-07-18 at 14 38 00" src="https://cloud.githubusercontent.com/assets/3276768/8761623/b034ca78-2d5a-11e5-92b5-28ee3bd8d8aa.png">

**Comment by remirobert** [comment]
I found something interesting, in the **support files** directory, in the **umbrella.h**, it's empty. Maybe i missed something, here.
